nuklear.h

```diff
@@ -294,27 +294,19 @@ extern "C" {
 #define NK_STRING_JOIN_DELAY(arg1, arg2) NK_STRING_JOIN_IMMEDIATE(arg1, arg2)
 #define NK_STRING_JOIN(arg1, arg2) NK_STRING_JOIN_DELAY(arg1, arg2)
 
-#ifdef _MSC_VER
-#define NK_UNIQUE_NAME(name) NK_STRING_JOIN(name,__COUNTER__)
```

Comment:
I'm pretty sure, these macros and defines shall stay and not be removed.

Reply:
Why the special treatment for MSVC? gcc and clang have `__COUNTER__` as well, but `__LINE__` is good enough for them? Wouldn't `__LINE__` work just as well for MSVC too? I may be missing something since I've never used MSVC.

Reply:
If it ain't broke don't fix it. Honestly I don't remember the reason. But I know that I got it from someone and he had a reason for it. However there is no reason to just delete something. Especially if you as well don't know the reason. Changing or removing something is always dangerous.

Reply:
@vurtun Agreed.